library/psa_crypto_core.h

```diff
@@ -64,6 +64,7 @@ typedef struct
         struct se
         {
             psa_key_slot_number_t slot_number;
+            size_t bits;
```

Comment:
Later you mention that having this as `size_t` is fragile. Why not set it to `uint64_t`?

Reply:
Any size other than `size_t` means a conversion and thus a range check somewhere, since it's `size_t` in the API and in the HAL. I did actually change the type, but in #197.